spec/rubocop/cop/graphql/ordered_fields_spec.rb

```diff
@@ -30,14 +30,15 @@ class UserType < BaseType
     end
   end
 
-  context "when each individual groups are alphabetically sorted" do
-    it "not registers an offense" do
-      expect_no_offenses(<<~RUBY)
+  context "when grouped fields are not alphabetically sorted" do
+    it "registers an offense" do
+      expect_offense(<<~RUBY)
```

Comment:
I had to change this spec, I know it kind of changes existing rubocop behavior today, that's why I was wondering if we really wanted this. But also, it feels confusing -- the rule should be named moreso `GraphQLOrderedFieldGroups`? Or maybe we have a configuration to allow that?

Reply:
I think we could make this part configurable: i.e. if `Groups: false` is off than we just ignore this check and pretend like it's the single group without breaks

Reply:
Yeah I realized that too. I can re-open and do that. The default can be `Groups: true` like you suggest 👌🏽